website/src/pages/grading/grade-output.tsx

```diff
@@ -1,113 +1,150 @@
+      </section>
 
-          {/* Skip / Submit controls */}
-          <div className="flex justify-center ml-auto">
-            <button
-              type="button"
-              className="mr-2 inline-flex items-center rounded-md border border-transparent bg-indigo-100 px-4 py-2 text-sm font-medium text-indigo-700 hover:bg-indigo-200 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
-            >
-              Skip
-            </button>
-            <button
-              type="button"
-              className="nline-flex items-center rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
-            >
-              Submit
-            </button>
+      {/* Info & controls */}
+      <section className="mb-8 p-4 rounded-lg shadow-lg bg-white flex flex-row justify-items-stretch ">
+        <div className="flex flex-col justify-self-start text-gray-700">
+          <div>
+            <span>
+              <b>Prompt</b>
+            </span>
+            <span className="ml-2">{tasks[0].id}</span>
           </div>
-        </section>
-      </div>
-    </>
-  );
-}
+          <div>
+            <span>
+              <b>Output</b>
+            </span>
+            <span className="ml-2">{tasks.length === 2 ? tasks[1].id : "Submit your answer"}</span>
+          </div>
+        </div>
 
-function RatingButton(props: { rating: number; active: boolean }): JSX.Element {
-  const activeClasses =
-    "inline-flex items-center mx-2 rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2";
-  const inactiveClasses =
-    "inline-flex items-center mx-2 rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow-sm hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2";
-  return (
-    <button type="button" className={props.active ? activeClasses : inactiveClasses}>
-      {props.rating}
-    </button>
+        {/* Skip / Submit controls */}
+        <div className="flex justify-center ml-auto">
+          <button
+            type="button"
+            className="mr-2 inline-flex items-center rounded-md border border-transparent bg-indigo-100 px-4 py-2 text-sm font-medium text-indigo-700 hover:bg-indigo-200 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
+          >
+            Skip
+          </button>
+          <button
+            type="button"
+            onClick={() => submitResponse(tasks[0])}
+            className="nline-flex items-center rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
```

Comment:
should this be `inline-flex` instead of `nline-flex`?

Reply:
I didn't catch this either in my first review.  It probably should be.  Will be fixed later.